keyboards/acheron/austin/keymaps/via/rules.mk

```diff
@@ -1 +1,2 @@
 VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = lite
```

Comment:
My understanding is that ` VIA_ENABLE = yes` enables Bootmagic Lite already.

https://github.com/qmk/qmk_firmware/blob/42d6270f28831e95d1cb9c14a7423d5b1d864d67/common_features.mk#L405-L411

Reply:
If its not already set, which in 99.9% of cases will be set at the keyboard level.